apps/herbatika/src/styles/tokens/_herbatika-spacing.css

```
@@ -36,6 +36,11 @@
   --spacing-homepage: var(--spacing-page-default);
   --spacing-homepage-lg: var(--spacing-page);
   --spacing-homepage-gap: var(--spacing-page-gap);
+  --spacing-homepage-promo-scroll-offset: calc(
+    var(--height-header-mobile) +
```

Stylelint (17.14.1)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)